ra/ra.go

```diff
@@ -116,6 +117,43 @@ func (ra *RegistrationAuthorityImpl) rateLimitPoliciesLoadError(err error) {
 	ra.log.Err(fmt.Sprintf("error reloading rate limit policy: %s", err))
 }
 
+// Run this to continually update the totalIssuedCount field of this
+// RA by calling out to the SA. It will run one update before returning, and
+// return an error if that update failed.
+func (ra *RegistrationAuthorityImpl) UpdateIssuedCountForever() error {
+	if err := ra.updateIssuedCount(); err != nil {
+		return err
+	}
+	go func() {
+		for {
+			_ = ra.updateIssuedCount()
+			time.Sleep(1 * time.Minute)
+		}
+	}()
+	return nil
+}
+
+func (ra *RegistrationAuthorityImpl) updateIssuedCount() error {
+	totalCertLimit := ra.rlPolicies.TotalCertificates()
+	if totalCertLimit.Enabled() {
+		now := ra.clk.Now()
+		count, err := ra.SA.CountCertificatesRange(
+			context.Background(),
```

Comment:
Does this use a background timeout so that we never timeout? If so could you add a comment to that effect.